src/jsifier.js

```diff
@@ -203,6 +203,7 @@ function ${name}(${args}) {
       'getWasmTableEntry',
       'runtimeKeepalivePush',
       'runtimeKeepalivePop',
+      'UTF8ToString',
```

Comment:
Why is this here? The comment on 199 says these are deps due to `makeDynCall` or `runtimeKeepalive*` but I don't think either of those generate strings?

Reply:
Sorry, I forget I added this.  Basically there are so many library functions that depend on this that manually adding it to `__deps` seems like more trouble that it was worth.

This is shortcut that automatically adds deps.  Hopefully we can do this for all functions one day.  See #19098.   For now I'm trying to keep this list small and focused.  I'll update the comment.